addons/default/visiosoft/advs-module/src/Http/Controller/Admin/AdvsController.php

```diff
@@ -328,42 +286,34 @@ public function replicate($advID)
 
     public function assetsClear(Filesystem $files, Application $application, Request $request)
     {
-        $directory = 'assets';
-        $files->deleteDirectory($directory = $application->getAssetsPath($directory), true);
-        echo "<div class='bar'></div>" . "<br>";
-        echo "<style>
-.bar {
-  width: 30%;
-  height: 20px;
-  border: 1px solid #2980b9;
-  border-radius: 3px;
-  background-image: 
-    repeating-linear-gradient(
-      -45deg,
-      #2980b9,
-      #2980b9 11px,
-      #eee 10px,
-      #eee 20px /* determines size */
-    );
-  background-size: 28px 28px;
-  animation: move .5s linear infinite;
-}
-
-@keyframes move {
-  0% {
-    background-position: 0 0;
-  }
-  100% {
-    background-position: 28px 0;
-  }
-}
+        $files->deleteDirectory($directory = $application->getAssetsPath('assets'), true);
+        echo "
```

Comment:
Why do we have the html inside the controller cant we call from a view ?

Reply:
moved it